src/Orleans/CodeGeneration/TypeUtils.cs

```diff
@@ -397,7 +397,7 @@ public static bool IsGrainClass(Type type)
         public static bool IsSystemTargetClass(Type type)
         {
             Type systemTargetType;
-            if (!TryResolveType("Orleans.Runtime.SystemTarget, OrleansRuntime", out systemTargetType)) return false;
+            if (!TryResolveType("Orleans.Runtime.SystemTarget", out systemTargetType)) return false;
```

Comment:
if this method's only purpose is for testing, I think it's fine to move it to the test.
Anyone know some context about this method?